nixos/roles/gitlab.nix

```diff
@@ -143,7 +143,7 @@ in
         databaseCreateLocally = fclib.mkPlatform true;
         databasePasswordFile = "${cfg.secretsDir}/db_password";
         initialRootPasswordFile = "${cfg.secretsDir}/root_password";
-        redisUrl = "redis://:${config.services.redis.servers."".requirePass}@localhost:6379/";
+        redisUrl = "redis://:${config.flyingcircus.services.redis.password}@localhost:6379/";
```

Comment:
needs to be read from file

Reply:
While I've never touched the gitlab module, I just noticed that you can configure another redis like this: https://docs.gitlab.com/omnibus/settings/redis/#using-an-alternate-local-redis-instance

Since the format is Ruby, one could also allow to read the contents of a file inside.